mypy/fastparse2.py

```diff
@@ -254,7 +253,7 @@ def translate_type_comment(self, n: ast27.stmt,
         ast27.BitXor: '^',
         ast27.BitAnd: '&',
         ast27.FloorDiv: '//'
-    }  # type: Final[Dict[typing.Type[AST], str]]
```

Comment:
Let's keep this since otherwise it may be unclear what the inferred type would be.